docs-source/content/userguide/introduction/architecture.md

```diff
@@ -88,3 +89,8 @@ The team developing the operator felt that these considerations provided adequat
 ### Network name predictability
 
 The operator uses services to provide stable, well-known DNS names for each server.  These names are known in advance of starting up a pod to run a server, and are used in the `ListenAddress` fields in the WebLogic Server configuration to ensure that servers will always be able to find each other.  This also eliminates the need for pod names or the actual WebLogic Server instance names to be the same as the DNS addresses.
+
+### Draining a node and PodDisruptionBudget
+A Kubernetes cluster administrator can [drain a Node](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/) for repair, upgrade, or scaling down the Kubernetes cluster.
+
+Beginning in version 3.2, the operator takes advantage of [PodDisruptionBudget](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets) feature offered by Kubernetes for high availability during a Node drain operation. The operator creates a PodDisruptionBudget (PDB) for each WebLogic cluster in the Domain namespace to limit the number of WebLogic Server pods simultaneously evicted when draining a node. The maximum number of WebLogic cluster's server pods evicted simultaneously is determined by the `maxUnavailable` field on the Domain resource. The `.spec.minAvailable` field of the PDB is calculated as `domain.spec.replicas - domain.spec.maxUnavailable`. For example, if you have a WebLogic cluster with three replicas and a maxUnavailable of 1, the `.spec.minAvailable` for PDB is set to 2. In this case, Kubernetes ensures that at least two pods for the WebLogic cluster's Managed Servers are available at any given time, and it only evicts a pod when all three pods are ready. For details about safely draining a node and PodDisruptionBudget concept, see [Safely Drain a Node](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/) and [PodDisruptionBudget](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/).
```

Comment:
takes advantage of [PodDisruptionBudget] -> takes advantage of the [PodDisruptionBudget]
and a maxUnavailable of 1 -> and a `maxUnavailable` of 1 (code font)
and PodDisruptionBudget concept -> and the PodDisruptionBudget concept

Reply:
Thanks, I have made the edits in 7801973.